install/kubernetes/cilium/templates/cilium-agent/role.yaml

```
@@ -0,0 +1,16 @@
+{{- if and .Values.agent (not .Values.preflight.enabled) .Values.serviceAccounts.cilium.create .Values.ingressController.enabled .Values.ingressController.secretsNamespace.name }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: cilium-secrets
+  namespace: {{ .Values.ingressController.secretsNamespace.name }}
```

nit: if we're quoting this value in `cilium-configmap.yaml` then let's do it everywhere:
```suggestion
  namespace: {{ .Values.ingressController.secretsNamespace.name | quote }}
```